src/test/java/pages/MainPage.java

```diff
@@ -0,0 +1,29 @@
+package pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MainPage extends BasePage {
+
+    @FindBy(className = "auth-bar__item--text")
+    public static WebElement entrance_Button;
```

Comment:
why do we need webelements to be static?

Reply:
no, fixed